source/module_base/global_function.cpp

```diff
@@ -2,52 +2,55 @@
 // AUTHOR : mohan
 //==========================================================
 #include "global_function.h"
-#include "global_file.h"
+
 #include "../src_parallel/parallel_common.h"
+#include "global_file.h"
 
 //==========================================================
 // USE FILE timer.h
 // ONLY :  output time after quit.
 //==========================================================
-#include "timer.h"
 #include "memory.h"
+#include "timer.h"
 
-#include <string>
 #include <fstream>
 #include <iostream>
+#include <string>
 namespace ModuleBase
 {
 namespace GlobalFunc
 {
 
 void NOTE(const std::string &words)
 {
-	return;
-	if(GlobalV::ofs_running)
-	{
-		//GlobalV::ofs_running << " *********************************************************************************" << std::endl;
-		GlobalV::ofs_running << " !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!" << std::endl;
-		GlobalV::ofs_running << " " << words << std::endl;
-		GlobalV::ofs_running << " !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!" << std::endl;
-	}
+    if (GlobalV::ofs_running)
+    {
+        // GlobalV::ofs_running << " *********************************************************************************"
+        // << std::endl;
+        GlobalV::ofs_running << " !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+                             << std::endl;
+        GlobalV::ofs_running << " " << words << std::endl;
+        GlobalV::ofs_running << " !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+                             << std::endl;
+    }
+    return;
```

Comment:
Change the place of "return;" would change output files. 
It seems like this function is falling into disuse.
Maybe keeping this function for doing nothing would be better?

Reply:
@dyzheng I suppose we can remove those "dead code".